tck/features/expressions/temporal/Temporal3.feature

```diff
@@ -27,12 +27,10 @@
 #
 
 #encoding: utf-8
-Feature: TemporalSelectAcceptance
+Feature: Temporal3 - Create Temporal Values from other Temporal Values
```

Comment:
I would call this "_Selecting_ Temporal Values from other Temporal Values"

Reply:
I thought about that. The original file also the term _selection_. Coming from the natural meaning of the word [_select_](https://www.merriam-webster.com/dictionary/select), the operation portraited in these scenarios allows to do more than just choosing components* to get a new temporal value. It allow to do overwrite components, too. If you wanted to refer to the relational projection, which I can see this operation has similarity too, I would not detour via SQL's SELECT, but go straight to it and speak of `Projecting Temporal Values from other Temporal Values`.

[*] As a side note, while I skimmed these scenarios, I wondered if `datetime`, `date`, and `time` are the only components one can select or if you can also do, say `{hour: other, minute: other}`. Probably not, there are no scenarios for that and `hour`, etc. are expected to be integers, I guess.

Reply:
```suggestion
Feature: Temporal3 - Project Temporal Values from other Temporal Values
```

Reply:
Just saw in [the CIP]( https://github.com/opencypher/openCypher/blob/master/cip/1.accepted/CIP2015-08-06-date-time.adoc#247-converting-and-composing-temporal-instant-values) and tried in the product that you can select `hour`, etc. So it is a gap in the scenarios. I will create a card about that.

Reply:
_Compose_ is another verb that is can be used here and is also used by the [CIP](https://github.com/opencypher/openCypher/blob/master/cip/1.accepted/CIP2015-08-06-date-time.adoc#247-converting-and-composing-temporal-instant-values).

```suggestion
Feature: Temporal3 - Compose Temporal Values from other Temporal Values
```